src/hisp/festim_models/mb_model.py

```diff
@@ -749,3 +753,156 @@ def make_DFW_mb_model(
     my_model.settings.stepsize = F.Stepsize(initial_value=1)
 
     return my_model, quantities
+
+
+def calculate_temperature_W(
+    x: float | NDArray, heat_flux: float, coolant_temp: float, thickness: float
+) -> float | NDArray:
+    """Calculates the temperature in the W layer based on coolant temperature and heat flux
+
+    Reference: [insert reference]
+
+    Args:
+        x: position in m
+        heat_flux: heat flux in W/m2
+        coolant_temp: coolant temperature in K
+        thickness: thickness of the W layer in m
+
+    Returns:
+        temperature in K
+    """
+    T_surface = 1.1e-4 * heat_flux + coolant_temp
+    T_rear = 2.2e-5 * heat_flux + coolant_temp
+    a = (T_rear - T_surface) / thickness
+    b = T_surface
+    return a * x + b
+
+
+def calculate_temperature_B(heat_flux: float, coolant_temp: float) -> float:
+    """Calculates the temperature in the boron layer based on coolant temperature and heat flux
+
+    Reference: [insert reference]
+
+    Args:
+        heat_flux: heat flux in W/m2
+        coolant_temp: coolant temperature in K
+
+    Returns:
+        temperature in K
+    """
+    T_rear_tungsten = (
+        2.2e-5 * heat_flux + coolant_temp
+    )  # boron layers based off of rear temp of W mbs
+    return 5e-4 * heat_flux + T_rear_tungsten
```

Comment:
boron temp model, including 5e-4 coeff from thermal contact resistivity, here: https://doi.org/10.1088/1741-4326/acd9d9
using equation (4) and the value R_c produced from JET experiments

Reply:
Can you provide the full calculation? Eq 4 just gives the expression of the global thermal resistnace of the Be layer.

It should be a simple enough calculation, we need to calculate `2.2e-5` and `5e-4` programmatically, and only hardcode numreical values we can find in the paper.

![image](https://github.com/user-attachments/assets/d76d2f58-6daa-4c5c-8069-02112a1c8902)

Reply:
Yes! Equation is the same model used in equation (5), where the 150, 250, 400 values are the tungsten temperatures (detailed on page 15 & in table 2

Reply:
5e-4 is a numerical value straight from the paper on page 14

Reply:
Noticed error in my code--should be taking T_surf instead of T_rear. will fix

Reply:
Could you write a test to catch this?